TheWarWithin/Priorities/RogueAssassination.simc

```diff
@@ -13,30 +13,49 @@ actions.precombat+=/stealth
 actions+=/stealth
 # Interrupt on cooldown to allow simming interactions with that
 actions+=/kick
-# Combined Energy Regen needed to saturate (TODO: estimate poisoned_bleeds better)
-actions+=/variable,name=regen_saturated,value=energy.regen_combined>15
-# AoE Damage over time abilities Check what the maximum Scent of Blood stacks is currently
-actions+=/variable,name=scent_effective_max_stacks,value=(spell_targets.fan_of_knives*talent.scent_of_blood.rank*2)>?20
-# We are Scent Saturated when our stack count is hitting the maximum
-actions+=/variable,name=scent_saturation,value=buff.scent_of_blood.stack>=variable.scent_effective_max_stacks
-# Check if we should be using our energy
-actions+=/variable,name=not_pooling,value=(dot.deathmark.ticking|dot.kingsbane.ticking|debuff.shiv.up)|(buff.envenom.up&buff.envenom.remains<=1)|energy.pct>=(40+30*talent.hand_of_fate-15*talent.vicious_venoms)|boss&fight_remains<=20
+# Combined Energy Regen needed to saturate
+actions+=/variable,name=regen_saturated,value=energy.regen_combined>30
```

Comment:
Does this threshold work well-enough with the recent Energy modeling changes?  My main concern is that this threshold wasn't met in the addon for newer 80s for a while.